WFP-VAM/prism-app
# Fix SimpleBoundaryDropdown would not show selected areas

### Description

Makes the SimpleBoundaryDropdown show selected areas when `setSelectedBoundaries` is defined.
Also remove unnecessary `react-window` package 



## How to test the feature:

- 'Go to' should still not show any area selected
- alert form and print ui should show the first areas selected and then use '...' for the rest

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
![image](https://github.com/WFP-VAM/prism-app/assets/80451946/28684847-d15c-4ca3-8a0e-8692ca768296)
